src/packages/fixednav/fixednav.tsx

```
@@ -79,19 +97,15 @@
             {list.map((item: any, index) => {
               return (
-                <div
-                  className={`${classPrefix}-list-item`}
-                  onClick={(event) => onSelect(item, event)}
-                  key={item.id || index}
-                >
-                  {React.isValidElement(item.icon) ? (
-                    item.icon
+                <>
+                  {item.num ? (
+                    <Badge value={item.num} top={8} right={6}>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)